src/stores/tokens.js

```diff
@@ -27,15 +36,32 @@ export const useTokensStore = defineStore('tokens', () => {
   }
 
   async function fetchAllTokens(queryParameters = null) {
-    allTokens.value = null
+    rawAllTokens.value = null
     const { data } = await axios.get(`${useRuntimeConfig().public.MIDDLEWARE_URL}${queryParameters || '/v2/aex9?by=name&direction=forward'}`)
-    allTokens.value = data
+    rawAllTokens.value = data
   }
 
   async function fetchListedTokens() {
     rawListedTokens.value = null
-    const { data } = await axios.get(`${useRuntimeConfig().public.DEX_BACKEND_URL}/tokens`)
-    rawListedTokens.value = data
+    const { data } = await axios.get(`${useRuntimeConfig().public.DEX_BACKEND_URL}/tokens/listed`)
```

Comment:
Additionally, I found out the `/listed` query works, so I could remove it from the frontend (from the adapter)